tests/maintel/test_mtcs.py

```diff
@@ -764,6 +765,21 @@ async def test_offset_xy_absorb(self) -> None:
             num=0,
         )
 
+    async def test_park_dome(self) -> None:
+        await self.mtcs.enable()
+        await self.mtcs.assert_all_enabled()
+
+        # Call the park_dome method
+        await self.mtcs.park_dome()
+
+        az_motion = await self.mtcs.rem.mtdome.evt_azMotion.aget()
```

Comment:
I think here you want to check that the commands where called with the parameters you expect and also the events where awaited in the way you expect.

Something like

```
self.mtcs.rem.mtdome.cmd_park.start.assert_awaited_with(timeout=self.mtcs.park_dome_timeout)
```

And something similar for the events you await.